src/send/mod.rs

```diff
@@ -116,15 +116,18 @@ fn handler(email: Email) -> Result<Json<Value>, Failure> {
         None
     };
 
-    match PROVIDERS.send(
-        email.to.as_ref(),
-        cc.as_ref(),
-        email.subject.as_ref(),
-        email.body.text.as_ref(),
-        html,
-        provider,
-    ) {
-        Ok(message_id) => Ok(Json(json!({ "messageId": message_id }))),
-        Err(_error) => Err(Failure(Status::InternalServerError)),
-    }
+    PROVIDERS
+        .send(
+            email.to.as_ref(),
+            cc.as_ref(),
+            email.subject.as_ref(),
+            email.body.text.as_ref(),
+            html,
+            provider,
+        )
+        .map(|message_id| Json(json!({ "messageId": message_id })))
+        .map_err(|error| {
+            println!("{}", error);
```

Comment:
Do you want the println here?

Reply:
I do (for now). While I was implementing this, stuff was broken for a while and I realised that nowhere was printing the error message. The most helpful thing I had to go on was `"Internal Server Error"`.

Note that soon we'll hopefully have proper logging in this repo, courtesy of #10. I know `println!` looks like a hack in the meantime (entirely reasonably of course, because it is one), but it's better than nothing.